inference-engine/tests/functional/plugin/cpu/single_layer_tests/interpolate.cpp

```diff
@@ -0,0 +1,190 @@
+using namespace CPUTestUtils;
+
+namespace CPULayerTestsDefinitions {
+
+typedef std::tuple<
+        LayerTestsDefinitions::InterpolateLayerTestParams,
+        CPUSpecificParams> InterpolateLayerCPUTestParamsSet;
+
+class InterpolateLayerCPUTest : public testing::WithParamInterface<InterpolateLayerCPUTestParamsSet>,
+                                     virtual public LayerTestsUtils::LayerTestsCommon, public CPUTestsBase {
+public:
+    static std::string getTestCaseName(testing::TestParamInfo<InterpolateLayerCPUTestParamsSet> obj) {
+        LayerTestsDefinitions::InterpolateLayerTestParams basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::tie(basicParamsSet, cpuParams) = obj.param;
+
+        std::ostringstream result;
+        result << LayerTestsDefinitions::InterpolateLayerTest::getTestCaseName(testing::TestParamInfo<LayerTestsDefinitions::InterpolateLayerTestParams>(
+                basicParamsSet, 0));
+
+        result << CPUTestsBase::getTestCaseName(cpuParams);
+
+        return result.str();
+    }
+
+protected:
+    void SetUp() {
+        LayerTestsDefinitions::InterpolateLayerTestParams basicParamsSet;
+        CPUSpecificParams cpuParams;
+        std::tie(basicParamsSet, cpuParams) = this->GetParam();
+
+        std::tie(inFmts, outFmts, priority, selectedType) = cpuParams;
+
+        LayerTestsDefinitions::InterpolateSpecificParams interpolateParams;
+        std::vector<size_t> inputShape;
+        std::vector<size_t> targetShape;
+        auto netPrecision   = InferenceEngine::Precision::UNSPECIFIED;
+        std::tie(interpolateParams, netPrecision, inputShape, targetShape, targetDevice) = basicParamsSet;
+
+        ngraph::op::v4::Interpolate::InterpolateMode mode;
+        ngraph::op::v4::Interpolate::CoordinateTransformMode coordinateTransformMode;
+        ngraph::op::v4::Interpolate::NearestMode nearestMode;
+        bool antialias;
+        std::vector<size_t> padBegin, padEnd;
+        double cubeCoef;
+        std:tie(mode, coordinateTransformMode, nearestMode, antialias, padBegin, padEnd, cubeCoef) = interpolateParams;
+
+        using ShapeCalcMode = ngraph::op::v4::Interpolate::ShapeCalcMode;
+        ShapeCalcMode shape_calc_mode = ShapeCalcMode::sizes;
+
+        auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+        auto params = ngraph::builder::makeParams(ngPrc, {inputShape});
+
+        auto constant = ngraph::opset3::Constant(ngraph::element::Type_t::i64, {targetShape.size()}, targetShape);
+
+        std::vector<float> scales(targetShape.size(), 1.0f);
+        auto scales_const = ngraph::opset3::Constant(ngraph::element::Type_t::f32, {scales.size()}, scales);
+
+        auto scalesInput = std::make_shared<ngraph::opset3::Constant>(scales_const);
+
+        auto secondaryInput = std::make_shared<ngraph::opset3::Constant>(constant);
+
+        ngraph::op::v4::Interpolate::InterpolateAttrs interpolateAttributes{mode, shape_calc_mode, padBegin,
```

Comment:
Could you please enable skipped interpolate single layer test? I said about https://github.com/openvinotoolkit/openvino/blob/0b61a7568f65a7b1cec879a102a3874315967c65/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp#L53
Ngraph interpolate reference was merged in the https://github.com/openvinotoolkit/openvino/pull/1412

Reply:
I already enable it in this branch. After merged, the change will reflect in master branch. thanks

Reply:
https://github.com/openvinotoolkit/openvino/blob/ce51603cf334867a5e0d9e2e14e3060550895263/inference-engine/tests/functional/plugin/cpu/shared_tests_instances/skip_tests_config.cpp